visual-java/src/main/java/com/saucelabs/visual/CheckOptions.java

```diff
@@ -49,6 +46,24 @@ public CheckOptions(
         null);
   }
 
+  private CheckOptions(CheckOptions.Builder builder) {
+    this.ignoreElements = builder.ignoreElements;
+    this.ignoreRegions = builder.ignoreRegions;
+    this.ignoreSelectors = builder.ignoreSelectors;
+    this.regions = builder.regions;
+    this.testName = builder.testName;
+    this.suiteName = builder.suiteName;
+    this.diffingMethod = builder.diffingMethod;
+    this.captureDom = builder.captureDom;
+    this.clipSelector = builder.clipSelector;
+    this.clipElement = builder.clipElement;
+    this.fullPageScreenshotConfig = builder.fullPageScreenshotConfig;
+    this.diffingOptions = builder.diffingOptions;
+    this.hideScrollBars = builder.hideScrollBars;
+    this.diffingMethodSensitivity = builder.diffingMethodSensitivity;
+    this.diffingMethodTolerance = builder.diffingMethodTolerance;
+  }
+
```

Comment:
It seems you cannot create `CheckOptions` manually, without using builder, with `diffingMethodSensitivity`/`diffingMethodTolerance`. Is this intended?

Reply:
I'm fine with only using builder for that or setters. I just thought that this was an omission.